src/WebJobs.Script.WebHost/Diagnostics/EtwEventGenerator.cs

```diff
@@ -52,5 +52,28 @@ public void LogFunctionExecutionEvent(string executionId, string siteName, int c
         {
             FunctionsEventSource.Instance.LogFunctionExecutionEvent(executionId, siteName, concurrency, functionName, invocationId, executionStage, (ulong)executionTimeSpan, success);
         }
+
+        public void LogFunctionDiagnosticEvent(LogLevel level, string resourceId, string operationName, string category, string regionName, string properties)
+        {
+            // Azure Monitor has no "Verbose" setting, so we map that to "Informational". We're controlling this logic here to minimize
+            // the amount of logic we have deployed with our monitoring configuration.
+            switch (level)
+            {
+                case LogLevel.Trace:
```

Comment:
Should this case be collapsed with the Information one below then, since the log statement is the same for all?

Reply:
Good point. Not sure why I set it up that way originally. I'll collapse it down and have them all log as Informational.